.github/PULL_REQUEST_TEMPLATE/new-feature.md

```diff
@@ -0,0 +1,48 @@
+## Description
+<!--
+Thank you so much for your PR!  To help us review your contribution, please
+consider the following points:
+
+- The PR title should summarize the changes, for example "Add new group argument for the
+  pair plot".  Avoid non-descriptive titles such as "Addresses issue #348". If your pull 
+  request addresses an issue, please use the pull request title to describe 
+  the issue and mention the issue number in the pull request description.
+
+- The description should provide at least 1-2 sentences describing the pull request
+  in detail (Why is this change required?  What problem does it solve?) and
+  link to any relevant issues. If modifying a plot, render your plot to inspect for changes 
+  and copy image in the pull request message on Github
+
+- Please prefix the title of incomplete contributions with [WIP] (to indicate a work in 
+  progress). WIPs may be useful to (1) indicate you are working on something to avoid 
+  duplicated work, (2) request broad review of functionality or API, or (3) seek collaborators.
+-->
+
+
+## Checklist
+<!-- Feel free to remove check-list items aren't relevant to your change -->
+
+- [ ] Does this PR follow the [official](https://github.com/arviz-devs/arviz/blob/master/CONTRIBUTING.md#pull-request-checklist) 
+      PR format?
+- [ ] Included a sample plot to visually illustrate your changes?
+- [ ] Do all the added functions and methods have docstrings?
+- [ ] Has added/updated unit tests where appropriate( using [pytest fixture pattern](
+      (https://docs.pytest.org/en/latest/fixture.html#fixture))?
+- [ ] Is code style correct (follows pylint, black guidelines)?
+- [ ] Has added documentation for new feature/functionality?
```

Comment:
suggested changes:

Included a sample plot to visually illustrate your changes? (only for plot-related functions)
Is the new feature properly documented with an example?